filebeat/harvester/processor/multiline.go

```diff
@@ -213,20 +215,23 @@ func (mlr *MultiLine) startNewLine(l Line) Line {
 	retLine := mlr.pushLine()
 	mlr.addLine(l)
 	mlr.ts = l.Ts
+	mlr.fields = l.Fields
 	return retLine
 }
 
 func (mlr *MultiLine) pushLine() Line {
 	content := mlr.content
 	sz := mlr.readBytes
+	fields := mlr.fields
```

Comment:
when merging multiple json events, which fields to we want to report? What if first one contains a timestamp?